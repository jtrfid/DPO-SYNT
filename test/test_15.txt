4 1

0 m
1 m
2 m
3 m

a c o

a 0
a 1
a 2
a 3