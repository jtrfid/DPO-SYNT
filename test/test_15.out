FSM: 
2 1

(0,0,0) m
(0,0,dead) m

a c o

a (0,0,dead)
a (0,0,dead)
REQ_FSM: 
1 1

(0,0,0) m

a c o


*******************************************************************
*********************All Inclusive Controller**********************
*******************************************************************
Y-States:
	Information State: {(0,0,0)}
	Transitions to:
		Z-State with Information State {{(0,0,0)},{a}} via Control Decision {a}
		Z-State with Information State {{(0,0,0)},{}} via Control Decision {}
	Information State: {(0,0,dead)}
	Transitions to:
		Z-State with Information State {{(0,0,dead)},{a}} via Control Decision {a}
		Z-State with Information State {{(0,0,dead)},{}} via Control Decision {}
Z-States:
	Information State: {{(0,0,0)},{}}
	Information State: {{(0,0,0)},{a}}
	Transitions to:
		Y-State with Information State {(0,0,dead)} via Event a
	Information State: {{(0,0,dead)},{}}
	Information State: {{(0,0,dead)},{a}}
	Transitions to:
		Y-State with Information State {(0,0,dead)} via Event a
*******************************************************************
*********************End All Inclusive Controller******************
*******************************************************************
DBTS for T_R:
******************************************************************************
********************Determined Bipartite Transition System********************
******************************************************************************
Y-States:
	Y_DBTS State: {(0,0,0)}
	Transitions to:
		Z_DBTS State {{(0,0,0)},{}}
Z-States:
	Z_DBTS State: {{(0,0,0)},{}}
******************************************************************************
********************Determined Bipartite Transition System********************
******************************************************************************
*******************************************************************
********************Control Simulation Relation********************
*******************************************************************
Y state pairs:
({(0,0,0)},{(0,0,dead)})
({(0,0,0)},{(0,0,0)})
Z state pairs:
({{(0,0,0)},{}},{{(0,0,dead)},{a}})
({{(0,0,0)},{}},{{(0,0,dead)},{}})
({{(0,0,0)},{}},{{(0,0,0)},{a}})
({{(0,0,0)},{}},{{(0,0,0)},{}})
*******************************************************************
********************Control Simulation Relation********************
*******************************************************************
RESULT SUPERVISOR:
******************************************************************************
********************Determined Bipartite Transition System********************
******************************************************************************
Y-States:
	Y_DBTS State: {(0,0,0)}
	Transitions to:
		Z_DBTS State {{(0,0,0)},{a}}
	Y_DBTS State: {(0,0,dead)}
	Transitions to:
		Z_DBTS State {{(0,0,dead)},{a}}
Z-States:
	Z_DBTS State: {{(0,0,0)},{a}}
	Transitions to:
		Y_DBTS State {(0,0,dead)} via Event a
	Z_DBTS State: {{(0,0,dead)},{a}}
	Transitions to:
		Y_DBTS State {(0,0,dead)} via Event a
******************************************************************************
********************Determined Bipartite Transition System********************
******************************************************************************
RESULT FSM:
2 1

(0,0,0) m
(0,0,dead) m

a c o

a (0,0,dead)
a (0,0,dead)
Press any key to continue...
